app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -480,13 +480,13 @@ void updatePlaybackSpeedButtonText() {
 
     void setupGUI() {
         setContentView(getContentViewResourceId());
-        sbPosition = findViewById(R.id.sbPosition);
+        sbPosition = findViewById(R.id.positionSeek);
```

Comment:
No idea about that. Because we are using an abstract class to set the view. Do you have any idea about how to do that?

Reply:
Oh, right. I did not realize that. I think the method always returns the same value though and can therefore be removed :)

Reply:
Please use ViewBinding here instead of accessing views directly. Maybe it would be easier to just bind to a protected variable in this class and then use it in the child class instead of managing two ViewBinding instances.